src/kernels/kernel.base.ts

```diff
@@ -330,9 +330,11 @@ export abstract class BaseKernel implements IKernel {
         if (!this.perceivedJupyterStartupTelemetryCaptured) {
             this.perceivedJupyterStartupTelemetryCaptured = true;
             sendTelemetryEvent(Telemetry.PerceivedJupyterStartupNotebook, stopWatch.elapsedTime);
-            executionPromise.finally(() =>
-                sendTelemetryEvent(Telemetry.StartExecuteNotebookCellPerceivedCold, stopWatch.elapsedTime)
-            );
+            executionPromise
+                .finally(() =>
+                    sendTelemetryEvent(Telemetry.StartExecuteNotebookCellPerceivedCold, stopWatch.elapsedTime)
+                )
+                .catch(noop);
```

Comment:
Why not use 'ignoreErrors'?

Reply:
For some reason I never liked that.

Reply:
It just feels a little dirty to wrap something so simple and native. Though i agree it saves an import.
& the downside of the extension to native promises is if someone else creates a similar extension that does something different. These aren't show stoppers, but enough to make me go `hmm, eew`